core/src/main/scala/org/apache/spark/sql/rapids/tool/store/AccumManager.scala

```diff
@@ -66,4 +66,13 @@ class AccumManager {
   def getMaxStageValue(id: Long): Option[Long] = {
     accumInfoMap.get(id).map(_.getMaxStageValue.get)
   }
+
+  /**
+   * Applies the function `f` to each entry in the accumInfoMap.
+   */
+  def applyToAccumInfoMap(f: (Long, AccumInfo) => Unit): Unit = {
+    accumInfoMap.foreach { case (id, accInfo) =>
+      f(id, accInfo)
+    }
+  }
```

Comment:
nit: you don't need `(Long, AccumInfo)` because accumInfo has the id, right?
Exposing the argument tuple (Long, AccumInfo) has a side effect of breaking "abstraction" because the caller is made aware that to access accumInfo, it requires a `<K,V>` pair.
It should be sufficient to:
```
  def applyToAccumInfoMap(f: AccumInfo => Unit): Unit = {
    accumInfoMap.value.foreach { accInfo =>
      f(accInfo)
    }
  }
```

Reply:
Thanks @amahussein. That is correct. Refactored the method definition.